contracts/rewards/Reward.sol

```diff
@@ -246,4 +257,13 @@ abstract contract Reward is IReward, ERC2771Context, ReentrancyGuard {
 
     emit NotifyReward(sender, token, epochStart, amount);
   }
+
+  /// @dev add new reward token to incentive.
+  function addRewardToken(address token) external {
```

Comment:
who can add reward token?

Reply:
everyone, which the new add token must be WhitelistedToken in Voter.

Reply:
in that case,  what's the point of adding the reward token address to Reward.sol?